build.gradle

```diff
@@ -26,8 +26,11 @@ task createClasspathManifest {
 }
 
 dependencies {
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.11.0'
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: '2.11.0'
+
     testCompile gradleTestKit()
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    testCompile group: 'junit', name: 'junit', version: '4.13'
```

Comment:
Is there a reason for using JUnit4 over JUnit5?

Reply:
Not really - I just went with whatever came with the template for Gradle plugins. Will see about upgrading in a new PR - this one has enough scope creep already :)